lib/gui.py

```diff
@@ -43,7 +44,8 @@ def focus_out(self, *args):
 
 
 DEFAULT_BG_COLOR = "#222"
-
+DEFAULT_WIDTH = 800
```

Comment:
I like these constants (not sure what they do as each module is different size, but I like them!)

Reply:
For now we have 4 windows in the same size so, I used them.